name.abuchen.portfolio/src/name/abuchen/portfolio/util/HolidayType.java

```diff
@@ -30,6 +30,353 @@ public LocalDate apply(LocalDate date)
         }
     }
 
+    private static class JewishCalendarDate
+    {
+        JewishCalendarDate(int day, int month, int year)
+        {
+            this.day = day;
+            this.month = month;
+            this.year = year;
+        }
+
+        // JewishCalendarDate(JewishCalendarDate date)
+        // {
+        // this.day = date.getDay();
+        // this.month = date.getMonth();
+        // this.year = date.getYear();
+        // }
```

Comment:
Avoid commented out code. In this case, I propose to delete it. If needed later, we can add such a constructor